etcdserver/auth/auth_test.go

```diff
@@ -73,14 +88,14 @@ func TestMergeUser(t *testing.T) {
 		},
 		{
 			User{User: "foo"},
-			User{User: "foo", Password: "$2a$10$aUPOdbOGNawaVSusg3g2wuC3AH6XxIr9/Ms4VgDvzrAVOJPYzZILa"},
```

Comment:
ok. we can drop the old one. then why do we want to merge with an empty password user? that case is unclear. it is unclear about if the password is overwritten by the new user or is preserved for the existing user since both of them are empty.